### DISCUSSION TITLE
Also send notifications to admins by email or via browser push notificition.

### DISCUSSION THREAD

**Discussion by woutersamaey** [btcpayserver/btcpayserver#2551] [Ideas / Feature Requests]
Since BTCPay Server has notifications, you only see them when you log in (which may not happen for some time).
It may be useful to receive notifications in your mailbox or through browser push notifications as well, if you would so choose to.

Creating this issue for myself, as I may look into building this.

**Comment by Kukks** [comment]
Please stop creating issues, these are feature requests and should follow the template

**Comment by woutersamaey** [comment]
@Kukks where is the Feature Request button / template / label? I'll clean up.

**Comment by Zaxounette** [comment]
> @Kukks where is the Feature Request button / template / label? I'll clean up.

![8jGJMo3Lr5](https://user-images.githubusercontent.com/51208677/119963782-c8d8e280-bfa8-11eb-8448-8e30959bad55.png)

![XP4ruSfRPf](https://user-images.githubusercontent.com/51208677/119963801-cd050000-bfa8-11eb-950f-18f35d3e82b0.png)

**Comment by woutersamaey** [comment]
@Zaxounette can I migrate an issue to a discussion or should I close and manually re-create?
